build/configs/rtl8721csm/Make.defs

```diff
@@ -161,10 +161,16 @@ CXXFLAGS = $(ARCHCXXFLAGS) $(ARCHWARNINGSXX) $(ARCHOPTIMIZATION) $(ARCHCPUFLAGS)
 
 ifeq ($(CONFIG_HAVE_CXX),y)
   CXXFLAGS += -std=c++11 -DCLOCK_MONOTONIC -D__TINYARA__
-  CXXFLAGS += -fexceptions -fcheck-new -frtti
+  CXXFLAGS += -fcheck-new -frtti
   CXXFLAGS += -pedantic -D_DEBUG -D_LIBCPP_BUILD_STATIC -ffunction-sections -fdata-sections -D__GLIBCXX__
 endif
 
+ifeq ($(CONFIG_HAVE_CXXINITIALIZE),y)
```

Comment:
There is a config CONFIG_LIBCXX_EXCEPTION. Let us set this flag based on this config and disable this config in loadable.